hasjob/static/js/app.js

```diff
@@ -184,6 +184,193 @@ window.Hasjob.PaySlider.prototype.resetSlider = function(currency) {
 };
 
 $(function() {
+  //Change site button to filter icon
+  $(".hg-site-nav-toggle").find("i").removeClass("fa-bars").addClass("fa-search");
+  $("#hg-sitenav").on("shown.bs.collapse", function() {
+    $(".hg-site-nav-toggle").find("i").removeClass("fa-search").addClass("fa-close");
+  });
+  $("#hg-sitenav").on("hidden.bs.collapse", function() {
+    $(".hg-site-nav-toggle").find("i").removeClass("fa-close").addClass("fa-search");
+  });
+
+  //Add Cancel button for dropdown filter in mobile
+  $("#job-filters-cancel").click(function(event) {
+    event.preventDefault();
+    $("#hg-sitenav").collapse("toggle");
+    $('body').removeClass('nav-open');
+  });
+  
+  var scrollheight = $("#hgnav").height() - $("#hg-sitenav").height();
+  $(window).scroll(function() {
+    if($(window).width() > 767) {
+      if ($(this).scrollTop() > scrollheight) {
+        $(".header-section").slideUp();
+      }
+      else{
+        $(".header-section").slideDown();
+      }
+    }
+  });
+
   window.Hasjob.JobPost.handleGroupClick();
   $(".pstar").off().click(window.Hasjob.JobPost.handleStarClick);
+
+  var getCurrencyVal = function() {
+    return $("input[type='radio'][name='currency']:checked").val();
+  };
+
+  var setPayTextField = function(){
+    var currencyLabel = 'Pay';
+    var equityLabel = '';
+    var payFieldLabel;
+
+    if ($('#job-filters-equity').is(":checked")) {
+      equityLabel += ' + ' + '%';
+    }
+    if (getCurrencyVal().toLowerCase() === 'na'){
+      currencyLabel = 'Pay';
+    } else {
+      currencyLabel = $('#job-filters-pmin').val() + ' - ' + $('#job-filters-pmax').val();
+    }
+    if (currencyLabel === 'Pay' && equityLabel !== '') {
+      payFieldLabel = 'Equity (%)';
+    } else {
+      payFieldLabel = currencyLabel + equityLabel;
+    }
+    $('#job-filters-pay-text').html(payFieldLabel);
+  };
+
+  $('#job-filters-equity').on('change', function(){
+    setPayTextField();
+  });
+
+  /* Hasjob.PayFilterParameters are assigned request parameters in index.html.
+    If Hasjob.PayFilterParameters is empty then assign default values. */
+  if (typeof window.Hasjob.PayFilterParameters === "undefined") {
+    window.Hasjob.PayFilterParameters = {
+      'currency': 'NA',
+      'pmin': 0,
+      'pmax': 10000000
+    };
+  }
+
```

Comment:
I don't think we need L249-L256. For instance, you can just do:

```javascript
Hasjob.currency || 'NA'
````
wherever you're using it.